udf-common

```diff
@@ -6,7 +6,7 @@ FS_TYPE='ext4'
 EPHEMERAL_MOUNT_POINT='/opt/indix'
 
 ROOT_DEVICE=$(df -h | grep "/$" | awk '{ print $1 }' | awk -F '/' '{ print $3 }' | sed 's/.$//g')
-EPHEMERAL_DEVICES=$(cat /proc/partitions | sed -n '1,2!p' | grep -v ${ROOT_DEVICE} | awk '{ print "/dev/"$4 }')
+EPHEMERAL_DEVICES=$(cat /proc/partitions | sed -n '1,2!p' | grep -v "ram[0-9]*" | grep -v ${ROOT_DEVICE} | awk '{ print "/dev/"$4 }')
```

Comment:
Given this is going to be used exclusively on AWS, why not query the [internal metadata endpoint](https://gist.github.com/philfreo/9ab42cae3f7e930f01882491b3c4e0d9#file-bootstrap_instance-sh-L109-L121) to get the list of ephemeral devices?

Reply:
It's nice to have, but I'd rather we remain vendor-agnostic.